docs/pages/machine-id/guides/ansible.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Using Teleport Machine ID with Ansible
+description: Using Teleport Machine ID with Ansible
+---
+
+In this guide, you will set up an Ansible playbook to run the OpenSSH client
+with a configuration file that is automatically managed by Machine ID.
+
+## Prerequisites
+
+You will need the following tools to use Teleport with Ansible.
+
+- [Teleport Enterprise](../../enterprise/introduction.mdx) or [Teleport Cloud](../../cloud/introduction.mdx) >= 9.0.0
+- `ssh` OpenSSH tool
+- `ansible` >= (=ansible.min_version=)
+- Optional tool `jq` to process `JSON` output
+
+In addition, if you already have not done so, follow the
```

Comment:
@russjones @ptgott important piece that guides should be self-contained. In this instance we are sending user to read a different guide - getting started first:

https://goteleport.com/docs/contributing/documentation/style-guide/#how-to-guides

> Do not break the reader's focus. Do not link the reader to another page—or even another section within the same page—when it is possible to include the same content in the current paragraph or section.

You can use partials to reduce duplication of the content internally.